Sources/Locker/Internal/LockerHelpers.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  LockerHelpers.swift
+//  
+//
+//  Created by Zvonimir Medak on 04.10.2021..
+//
+
+import Foundation
+import LocalAuthentication
+import System
+
+@available(macOS 10.13.2, *)
+class LockerHelpers: LockerHelpable {
+
+    // MARK: - Public properties
+
+    static var biometricsSettingsChanged: Bool {
+        var biometricsSettingsChangedStatus = false
+
+        let oldDomainState = LockerHelpers.savedLAPolicyDomainState
+        let newDomainState = LockerHelpers.currentLAPolicyDomainState
+
+        // Check for domain state changes
+        // For deactivated biometrics, LAContext in validation will return nil
+        // storing that nil and comparing it to nil will result as `isEqual` NO
+        // even data is not actually changed.
+        let biometricsDeactivated: Bool   = (oldDomainState == nil)  || (newDomainState == nil)
+        let biometricSettingsDidChange = oldDomainState?.elementsEqual(newDomainState!) ?? false
+        if (biometricsDeactivated && biometricSettingsDidChange) {
+            biometricsSettingsChangedStatus = true
+
+            LockerHelpers.setLAPolicyDomainState(with: newDomainState)
+        }
+
+        return biometricsSettingsChangedStatus
+    }
+
+    static var configureBiometricsAuthentication: BiometricsType {
+        let canUse = LAContext().canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
+
+        if canUse && LockerHelpers.canUseAuthenticationWithFaceID {
+            return .faceID
+        } else if canUse {
+            return .touchID
+        } else {
+            return .none
+        }
+    }
+
+    static var keyLAPolicyDomainState: String {
+        String(format: "%@_UserDefaultsLAPolicyDomainState", locale: nil, LockerHelpers.kBundleIdentifier)
```

Comment:
```suggestion
        "\(LockerHelpers.kBundleIdentifier)_UserDefaultsLAPolicyDomainState"
```
Change to swift notation on other places as well. Other than that, please remove `k` prefix in all constants as this is something only used in ObjC :)